README.md

```diff
@@ -129,7 +141,37 @@ This command has two modes: bounded and unbounded.
 ##### Unbounded
 If only a start ledger is provided, then the command runs in an unbounded fashion starting from the provided ledger. In this mode, the Stellar Core connects to the Stellar network and processes new changes as they occur on the network. Since the changes are continually exported in batches, this process can be continually run in the background in order to avoid the overhead of closing and starting new Stellar Core instances.
 
-## Schema
+#### export_orderbooks
+
+```bash
+> stellar-etl export_orderbooks --start-ledger 1000 \
+--end-ledger 500000 --output exported_orderbooks_folder/
+```
+
+This command exports orderbooks within the provided ledger range. Since exporting complete orderbooks at every single ledger would require an excessive amount of storage space, the output is normalized. Each batch that is exported contains multiple files, namely: `dimAccounts.txt`, `dimOffers.txt`, `dimMarkets.txt`, and `factEvents.txt`. The dim files relate a data structure to an ID. `dimMarkets`, for example, contains the buying and selling assets of a market, as well as the ID for that market. That ID is used in other places as a replacement for the full market information. This normalization process saves  a significant amount of space. The `factEvents` file connects ledger numbers to the offer IDs that were present at that ledger.
+
+Orderbooks are exported in batches of a size defined by the `batch-size` flag. By default, the batch-size parameter is set to 64 ledgers, which corresponds to a five minute period of time. This batch size is convenient because checkpoint ledgers are created every 64 ledgers. Checkpoint ledgers act as anchoring points for the nodes on the network, so it is beneficial to export in multiples of 64.
```

Comment:
What's an "anchoring point" for a node? It's worth explaining concepts like this, at least briefly, in documentation.